examples/src/main/java/com/mapbox/navigation/examples/core/MapboxVoiceActivity.kt

```diff
@@ -389,6 +394,7 @@ class MapboxVoiceActivity : AppCompatActivity(), OnMapLongClickListener {
         if (::mapboxNavigation.isInitialized) {
             mapboxNavigation.registerRoutesObserver(routesObserver)
             mapboxNavigation.registerLocationObserver(locationObserver)
+            mapboxNavigation.registerRouteProgressObserver(voiceInstructionsDownloadTrigger)
```

Comment:
```suggestion
mapboxNavigation.registerRoutesObserver(voiceInstructionsDownloadTrigger)            mapboxNavigation.registerRouteProgressObserver(voiceInstructionsDownloadTrigger)
```

I haven't tested this activity, but voiceInstructionsDownloadTrigger implements RouteProgressObserver and RoutesObserver

Reply:
```suggestion
            mapboxNavigation.registerVoiceInstructionsTriggerObserver(voiceInstructionsDownloadTrigger)
```

Sorry it may be this..